src/main/java/webdriver/DriverFactory.java

```diff
@@ -0,0 +1,35 @@
+package webdriver;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeDriverService;
+
+import java.util.concurrent.TimeUnit;
+
+public class DriverFactory {
+    private static final String CHROMEDRIVER_EXE_PATH = "src/main/resources/chromedriver.exe";
+    public static final int IMPLICITLY_WAIT = 10;
+
+    public enum BrowserType {FIREFOX, CHROME, EDGE, IE}
+
+    public static WebDriver getDriver(BrowserType type) {
+        WebDriver driver = null;
+        switch (type) {
+            case CHROME:
+                driver = getChromeDriver();
+                break;
+            default:
+                driver = getChromeDriver();
```

Comment:
In what case default block is executed? Do you think that chrome browser should be selected if smth incorrect is passed to the method? I think you should throw an exception.